### DISCUSSION TITLE
All Wallet shows 0

### DISCUSSION THREAD

**Discussion by mongchanghsi** [reown-com/appkit#2549] [General]
I have added in a WalletConnect ID but in my localhost, All Wallet shows 0. The following is the configuration I have placed.

```
export const wagmiConfig = defaultWagmiConfig({
  projectId,
  chains,
  metadata,
});

export const web3Modal = createWeb3Modal({
  wagmiConfig,
  projectId,
  defaultChain: mainnet,
});
```

**Comment by rtomas** [comment]
Please, Can you explain better the situation and share some code snippets or a minimal working example. ?
Also you can test all our features in -> https://lab.web3modal.com/
